nomad/config.go

```diff
@@ -493,6 +493,15 @@ func (c *Config) VaultDefaultIdentity() *structs.WorkloadIdentity {
 	return workloadIdentityFromConfig(c.VaultConfig.DefaultIdentity)
 }
 
+func (c *Config) VaultIdentiyConfig(cluster string) *structs.WorkloadIdentity {
```

Comment:
```suggestion
func (c *Config) VaultIdentityConfig(cluster string) *structs.WorkloadIdentity {
```

Reply:
I managed to get a through 1.6 cycle without committing a single `NodePoo` or `NodePoll` but it seems like `identity` is the real challenge 😅